Parse/src/main/java/com/parse/ParseUser.java

```diff
@@ -1255,8 +1240,10 @@ public ParseUser then(Task<Void> task) throws Exception {
     }
   }
 
-  /* package */ Task<Void> linkWithAsync(
-      final String authType, final Map<String, String> authData, final String sessionToken) {
+  private Task<Void> linkWithAsync(
+      final String authType,
+      final Map<String, String> authData,
+      final String sessionToken) {
     return taskQueue.enqueue(new Continuation<Void, Task<Void>>() {
```

Comment:
Maybe enforce authType and authData to be non-nil?
Also, not sure why we need this method, since we don't seem to be using it much often.